Makefile

```diff
@@ -3,6 +3,17 @@ IMG ?= quay.io/ecosystem-appeng/dbaas-e2e-test-harness:latest
 DIR := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
 OUT_FILE := "$(DIR)dbaas-e2e-test-harness"
 
+install-tools:
```

Comment:
Move these further down so that `build` is the first target and thereby the default target.

Reply:
Done.